docs/platforms/react-native/migration/v6-to-v7.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: Migrate from 6.x to 7.x
+sidebar_order: 99
+draft: true
+
+description: "Learn how to migrate from version 6 to version 7 of the Sentry React Native SDK"
+---
+
+The main goal of version 6 of the Sentry React Native SDK, is to bring compatibility with Sentry JavaScript version 9. This version is introducing breaking changes because of the JavaScript SDK dependency upgrade to version 9 and because we've removed deprecated APIs and restructured npm package contents.
```

Comment:
```suggestion
The main goal of version 7 of the Sentry React Native SDK, is to bring compatibility with Sentry JavaScript version 9. This version is introducing breaking changes because of the JavaScript SDK dependency upgrade to version 9 and because we've removed deprecated APIs and restructured npm package contents.
```